test/quantization/test_quantize_fx.py

```diff
@@ -1330,9 +1330,23 @@ def forward(self, x):
         }
         for dim, quant_type in options:
             model = self.checkGraphModeFxOp(
-                Conv(dim), self.img_data_dict[dim], quant_type,
+                ConvWrapper(dim), self.img_data_dict[dim], quant_type,
                 quantized_nodes[dim])
 
+    @skipIfNoFBGEMM
+    def test_conv_functional_bias_not_observed(self):
```

Comment:
optional: can we merge this test with previous test and call it "test_conv"? we have "test_linear" that checks all combinations of is_module/is_not_module and has_rleu/doesn't have relu and relu module/functional. I think that would be a more exhaustive testing.

Reply:
hmm, that was intentional, there are benefits when each test case is separate.  This allows for faster debugging when chasing down an individual issue (so there is no need to comment out the unrelated test cases in the large loop).  Perhaps we can align on a way to both share code and also test one thing at a time, lmk your preference.

Reply:
I think we should do exhaustive tests on what we want to support first. in this particular case, the problem is that we did not test functional conv before. Testing for special cases that we encounter in model might be too ad hoc and it will be hard to know what is covered and what is not in the future.

I think we can first make sure our unit test cover all possible modules/ops etc. we want to support, and then if there is any corner case we can cover with these ad hoc tests (typically should be some interaction between ops + qconfig instead of one op). one such example is here: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_fx.py#L538

Reply:
regarding separating test cases I think as long as there is no code duplication it should be OK.

Reply:
sure, I made the functional test more comprehensive.

Regarding no code duplication, I think it's good to optimize for in general, but code readability and ease of use is something to consider as well.  Some of the current test utils seem to bias on the "minimize code reuse" side while sacrificing readability, at least for people who were not the code authors.  Maybe in the future we can consider on ways to do both.

Reply:
sure sounds good